cli/src/semgrep/commands/ci.py

```diff
@@ -358,6 +373,12 @@ def ci(
         exclude = (*exclude, *yield_exclude_paths(excludes_from_app))
         assert config  # Config has to be defined here. Helping mypy out
         start = time.time()
+
+        if scan_handler and not scan_handler.enabled_products:
+            raise SemgrepError(
+                'No products are enabled for this organization. Please enable a product in the "Deployment" settings tab of the Semgrep Cloud Platform or reach out to support@semgrep.com for assistance.'
```

Comment:
```suggestion
                'No products are enabled for this organization. Please enable a product in the Settings > Deployment tab of the Semgrep Cloud Platform or reach out to support@semgrep.com for assistance.'
```

Reply:
not wedded to this suggestion in particular but this is how our docs refer to the settings selections

Reply:
I updated the language because I like this better